projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -776,8 +768,10 @@ function maybe_close_comments( $default_comments_open, $post_id ) {
 	if ( ! $default_comments_open || ! $post_id ) {
 		return $default_comments_open;
 	}
+
+	$is_post = false;
```

Comment:
I am curious about this variable being set here...

I almost thought it was used as a global in `class-jetpack-memberships.php`... It feels a bit misleading.

Also, It looks like `Jetpack_Memberships::user_can_view_post` [does not accept parameters](https://github.com/Automattic/jetpack/pull/30807/files#diff-6c4e5f6a52aca3eb00a4cc6c302da733d70d8d26b61d4ba9e4ec2a8141add6e3R469)... 

Am I missing something here 🤔

Reply:
Thanks, fixed in [53ed6ba](https://github.com/Automattic/jetpack/pull/30807/commits/53ed6baca5be6d2e6bc078872cbe8b0ba413fe34)